types.d.ts

```diff
@@ -0,0 +1,38 @@
+declare module "react-barcode" {
+  declare namespace ReactBarcode {
```

Comment:
remove `declare` below module, 
TS error: A 'declare' modifier cannot be used in an already ambient context.

Reply:
That's more elegant. I'll change namespace declaration to a module one!
I'm still so new to Typescript. Thanks for letting me know!